tsconfig.json

```diff
@@ -1,13 +1,21 @@
 {
   "compilerOptions": {
     "outDir": "./dist/",
+    "rootDir": "./",
+    "target": "es5",
     "module": "commonjs",
+    "moduleResolution": "node",
```

Comment:
Why do we need to add so many new flags?

For example, we are not using decorators and enums? I'm not sure why we need to add flags for those. 
isolatedModules is also `false` by default.